pipeline/continuation/model.py

```diff
@@ -0,0 +1,123 @@
+"""
+Continue the training pipeline with an existing model.
+"""
+import argparse
+from pathlib import Path
+from pipeline.common.downloads import stream_download_to_file, location_exists
+from pipeline.common.logging import get_logger
+from pipeline.common import arg_utils
+
+logger = get_logger(__file__)
+
+potential_models = (
+    "model.npz.best-chrf.npz",
+    "model.npz",
+    "model.npz.best-bleu-detok.npz",
+    "model.npz.best-ce-mean-words.npz",
+    "final.model.npz.best-chrf.npz",
+    "model.npz.optimizer.npz",
+)
+
+potential_decoders = [
+    "model.npz.best-chrf.npz.decoder.yml",
+    "model.npz.best-bleu-detok.npz.decoder.yml",
```

Comment:
I don't think we need all of these. If we want to continue training with the same corpus, we need only `model.npz.yml` and I think `model.npz.optimizer.npz`. If we want to initialize to fine-tune we need `final.model.npz.best-chrf.npz`. And for "use" mode it's two `final*` files (we should double check this). It probably makes sense to copy everything for preemption but it's a different mechanism. Here, when we just want to replace model training with getting the final artifacts, we need only `final*` files.

Reply:
Ok, so I looked at this a bit, and I think the correct thing to do here is to move the `final.model.npz.best-chrf.*` to the top of the list, as only one file is output, but the others are searched. This should make it robust for old archived models. It should be harmless to include old one, and then it can help not break for re-using old models.